mod/fabric/build.gradle

```diff
@@ -30,6 +30,11 @@ dependencies {
 	modCompileOnly "maven.modrinth:journeymap:5JbcGXLn"
 	// https://modrinth.com/mod/xaeros-minimap/version/23.6.2_Fabric_1.18.2 (23.6.2 fabric)
 	modCompileOnly "maven.modrinth:xaeros-minimap:Jwydpps9"
+
+
+
+	// https://modrinth.com/mod/modmenu/version/3.2.5 (3.2.5 fabric)
+	modCompileOnly "maven.modrinth:modmenu:nVxObSbX"
```

Comment:
Formatting (consecutive newlines)

Reply:
Those are deliberate, to keep them separated from the mapping mod dependencies, in the same way the mapping mod dependencies have three spaces between them and the common-project dependencies. I can change this if need be, but it's intended to keep things clean and partitioned.